nwb_conversion_tools/baserecordingextractorinterface.py

```diff
@@ -42,20 +42,14 @@ def get_metadata_schema(self):
 
     def get_metadata(self):
         """Auto-fill as much of the metadata as possible. Must comply with metadata schema."""
-        out = super().get_metadata()
-        out['properties'].update(
+        return dict(
             Ecephys=dict(
-                ElectricalSeries=dict(
-                    name='ElectricalSeries',
-                    description='raw acquired data'
-                ),
-                Device=dict(
-                    name='device',
-                    description='ecephys probe'
-                )
+                Device=[],
+                ElectrodeGroup=[],
+                Electrodes=[],
+                ElectricalSeries=dict()
```

Comment:
I mentioned there currently isn't a way to force spikeextractors to automatically write default metadata since it relies on the entire metadata dictionary being passed as None. Well, if we just set the internals to empty lists it will have the same effect by autopopulating each required field with the defaults over there (which are similar to the defaults that were removed here).